src/main/java/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction.java

```diff
@@ -624,6 +656,35 @@ public void doSaveNote(final StaplerRequest req, final StaplerResponse rsp) thro
         }
     }
 
+    // ---------------------------------------------------------------------------
+    /** Change queue order (item position) */
+    @Restricted(NoExternalUse.class) // used by jelly
+    @RequirePOST
+    public void doChangeQueueOrder(final StaplerRequest req, final StaplerResponse rsp)
+            throws IOException, ServletException {
+        Jenkins.get().checkPermission(UNLOCK);
```

Comment:
Out of curiosity: why bind it to the `UNLOCK` permission instead of the `STEAL` permission?

Stealing locks feels more manipulative than unlocking. And checking queue orders feels equally manipulative.

Or is there a clear rationale for using `UNLOCK`?

Reply:
POV from my company. Not everyone have this priority.  And resource stolen might be done by more users (by us) . The permissions to unlock has only few peoples (like admins). An I want to have this action under control.  Otherwise it might happened   that operator change the order in weird way, and need to spend time to analyse,  why the production branches takes soo long. Now it will be necessary to contact my group, when some operator need to execute the scope early. Anyway,  I think I need to spend tome, to add new permissions for this action. In that case,  the administrator can configure it as proposed by own rules

Reply:
@PayBas new permission added.
May you pls re-test / re-review tha cange agin. thx